R/broom_wrapper.R

```diff
@@ -692,7 +692,11 @@ prediction_survfit <- function(df, ...){
 #' tidy after generating survfit
 #' @export
 do_survfit <- function(df, time, status, ...){
-  ret <- df %>% build_model(model_func = survival::survfit, formula = survival::Surv(time, status) ~ 1, ...) %>% broom::tidy(model)
+  # need to compose formula with non-standard evaluation.
+  # simply using time and status in formula here results in a formula that literally looks at
+  # "time" columun and "status" column, which is not what we want.
+  fml <- as.formula(paste0("survival::Surv(", substitute(time), ",", substitute(status), ") ~ 1"))
```

Comment:
I guess this code causes error for columns with spaces. If so, please use back tick